hack/run-ci-e2e-test.sh

```diff
@@ -178,13 +178,17 @@ if [[ "$TEST" = "all" || "$TEST" = "upgrade" ]]; then
 fi
 
 if [[ "$TEST" = "upgrade-setup" ]]; then
+  # TODO: Remove this after adding it to the appropriate release jobs
```

Comment:
Are all the TODO comments necessary if we will revert the whole commit in thew 4.15 branch? They'd unnecessarily linger in 4.14

Reply:
If you want to revert this, I suggest introducing it as a separate PR so that we can just revert the PR.

Reply:
This needs to go in at the same time as the previous commit, as this will not work on 4.14 without it. But I have split the rest of the commits off into a separate PR: https://github.com/openshift/windows-machine-config-operator/pull/1733, as they are just general test improvements.